go/registry/api/api.go

```diff
@@ -80,6 +80,10 @@ var (
 	// register a runtime with bad Capabilities.TEE.Hardware.
 	ErrBadCapabilitiesTEEHardware = errors.New("registry: bad capabilities.TEE.Hardware")
 
+	// ErrDebugEnclave is the error returned when a node tries to register
+	// runtimes with enclaves running in debug mode.
+	ErrDebugEnclave = errors.New("registry: enclave running in debug mode not allowed")
```

Comment:
Is the same error used if the node tries to register a production enclave but the network is running in debug mode? In this case the error description (and doc comment) should probably be updated.